src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.Designer.cs

```diff
@@ -1473,6 +1473,15 @@ public class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Getting Package Source Mappings....
+        /// </summary>
+        public static string PackageSourceMappingOptions_OnActivated {
+            get {
+                return ResourceManager.GetString("PackageSourceMappingOptions_OnActivated", resourceCulture);
```

Comment:
See `OnActivate` comment if you're curious where this string actually gets used.